lib/console/vmstat_utils.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic functions for testing vmstat.
+# Maintainer: QE Core <qe-core@suse.de>
+
+package console::vmstat_utils;
```

Comment:
I wonder, why do you separate out this file and not keep the implementation in the only test module file that currently uses the functionality? Commonly lib/ should be used for functionality that is used in more than one test module

Reply:
This was a suggestion by other colleague to keep only main function in test module file and remaining functionality in separate file commonly /lib.

Reply:
ok, can you explain why? Sorry, I did not see that suggestion

Reply:
It was during pair programming session.